src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(func, num) {
+      this.result = func(this.result, num);
+
+      return this;
+    },
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      return b === 0 ? 'ERROR! You can`t divide by zero!' : a / b;
```

Comment:
Better to use `throw` statement to throw an exception 👍
```suggestion
      return b === 0 ? throw new Error('ERROR! You can`t divide by zero!') : a / b;
```